resources/ts/Components/Integrations/Detail/CredentialsAuthClients.tsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import { Heading } from "../../Heading";
+import { useTranslation } from "react-i18next";
+import { Integration } from "../../../Pages/Integrations/Index";
+import { StatusLight } from "../../StatusLight";
+//import { ButtonPrimary } from "../../ButtonPrimary";
+import { CopyText } from "../../CopyText";
+import { ActivationFlow } from "../../ActivationFlow";
+import { IntegrationStatus } from "../../../types/IntegrationStatus";
+import { Credentials } from "./Credentials";
+
+type Props = {
+  email: string;
+  id: Integration["id"];
+  status: Integration["status"];
+  subscription: Integration["subscription"];
+  type: Integration["type"];
+} & Credentials;
```

Comment:
You can also use `Pick` I think to make this new type.
```suggestion
type Props = Pick<Integration, 'email' | 'id' | 'status' | 'subscription' | 'type' > & Credentials;
```